adapter/ryu/omniui/omniui.py

```diff
@@ -18,8 +18,24 @@
 from ryu.lib import ofctl_v1_3
 from ryu.lib.dpid import dpid_to_str
 from ryu.topology.api import get_switch, get_link
+import requests
+import subprocess
+from operator import attrgetter
+from ryu.ofproto.ether import ETH_TYPE_LLDP, ETH_TYPE_IPV6
+from ryu.lib import hub
+from ryu.lib.packet import *
+from ryu.topology import event, switches
+
+global controllerName
+controllerName = 'DEFAULT'
+global coreIP
+coreIP = '127.0.0.1'
+global corePort
+corePort = '5567'
```

Comment:
To improve the flexibility further, please load these value from config file so users don't have to modify the code to apply new configuration.